Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -87,13 +87,16 @@ public SiteExtensionManager(IContinuousJobsManager continuousJobManager, ITrigge
                             .OrderByDescending(p => p.DownloadCount);
             }
 
-            using (tracer.Step("Convert search result to SiteExtensionInfos"))
+            using (tracer.Step("Convert search result to SiteExtensionInfos with max concurrent requests: {0}", System.Environment.ProcessorCount))
             {
+                // GetResourceAsync is not thread safe
```

Comment:
Is the comment referring to `GetResourceAndValidateAsync` or a different method called `GetResourceAsync`?